src/docker-osx-dev

```diff
@@ -1031,6 +1034,23 @@ function install {
 }
 
 #
+# Fetch the latest docker-osx-dev script, and then run install
+#
+function update_self {
+  log_info "Updating to the latest docker-osx-dev"
+  local readonly current_file=$(which docker-osx-dev)
+  local readonly backup_file=$current_file.old
```

Comment:
Probably a good idea to wrap `$current_file.old` in quotes, in case the path has spaces in it.